subsys/bluetooth/mesh/access.c

```diff
@@ -1892,7 +1893,16 @@ static int comp_set(const char *name, size_t len_rd, settings_read_cb read_cb,
 
 	return 0;
 }
-BT_MESH_SETTINGS_DEFINE(comp, "cmp", comp_set);
+BT_MESH_SETTINGS_DEFINE(comp, "cmp0", comp_p0_set);
```

Comment:
Seems this breaks backward compatibility. Could you apply the same trick that Pavel did for virtual address support?
https://github.com/zephyrproject-rtos/zephyr/pull/57878/files#diff-007d797c63a5cbfd92a1922fc7fe3e40502988f252054ebeb6da1d77e5f25e87R334
https://github.com/zephyrproject-rtos/zephyr/pull/57878/files#diff-fced093b1243729a3f60760434e674f6c075f19c42b1e71aa44b8c1eb63721eeR1742

Reply:
But if I reinstate the `"cmp"` name this is not an issue, right? I did not think of the backwards compabillity when I did this, but if it causes issues changing the name I can avoid changing the name.

Reply:
In general, I like your change. Page index is a good idea. The only issue is backward compatibility.

Reply:
FYI. There is no issue with backward compatibility. This code is part of mesh-1.1 features. Mesh-1.1 is still experimental. No one should have been using it in production yet. So it is absolutely fine to change the way CDP0 is stored.